coretests/tests/testerplugin.py

```diff
@@ -48,8 +48,26 @@ def functionalTests():
     testBrowserAddWmts.addStep('Try to add layer "112 Par satellite" from the Browser to QGIS canvas. Check that QGIS prompts for date and after setting a date layer added to canvas and visible.')
     testBrowserAddWmts.setCleanup(lambda: _removeWmtsConnection())
 
+    # Verify if message log shows errors
+    logTest = Test("Verify in-app message log has no errors for default install. QGIS-62")
+    logTest.addStep("Open log messages panel",
+                    prestep=lambda:_openLogMessagesDialog())
```

Comment:
Better to make it fully automated step by defining step function. Also there is a native API call to open message log, so better to use it instead of custom function.